common/enclave/common-output.edl

```diff
@@ -0,0 +1,11 @@
+/*
+ * Copyright IBM Corp. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+enclave {
+    untrusted {
+        void ocall_print_string([in, string] const char *str);
+    };
+};
```

Comment:
what about renaming `common-output.edl` to just `logging.edl`

Reply:
no strong opinion on that. Done!